installers/linux/packages/src/Versions.elm

```diff
@@ -0,0 +1,14 @@
+module Versions exposing (architectures, versions)
+
+
+architectures : List { lamderaName : String, debianName : String }
+architectures =
+    [ { lamderaName = "x86_64", debianName = "amd64" }
+    , { lamderaName = "arm64", debianName = "aarch64" }
+    ]
+
+
+versions : List { lamderaVersion : String, elmVersion : String }
+versions =
+    [ { lamderaVersion = "1.3.2", elmVersion = "0.19.1" }
```

Comment:
Given we're already in scripts land, lets read the version number from https://github.com/lamdera/compiler/blob/lamdera-next/extra/Lamdera/Version.hs#L14 – even if it's just a dumb string read and split.

Reply:
The idea was that we'd add all the versions we care about from the list. I could read it from `Version.hs` but I think it makes more sense to have it explicit, because we want to keep creating them for older versions even when we upgrade

Reply:
Oh? Does the deb publishing require us to keep posting entries for old versions when we do entries for new ones..?

Reply:
Hrm okay, until now the premise has been nobody should be using older Lamdera versions, but that was mainly due to the platform always being on the latest – but I guess in the context of general Elm tooling that changes.